geomesa-tools/conf/application.conf

```diff
@@ -1,46 +1,41 @@
-geomesa = {
-  sfts = [
-    {
-      type-name = "example-csv"
+geomesa {
+  sfts {
+    "example-csv" = {
       attributes = [
-        {name = "id", type = "Integer", index = false},
-        {name = "name", type = "String", index = true},
-        {name = "age", type = "Integer", index = false},
-        {name = "lastseen", type = "Date", index = true},
-        {name = "friends", type = "List[String]", index = true},
-        {name = "geom", type = "Point", index = true, srid = 4326, default = true}
+        {name = "id",       type = "Integer",      index = false},
+        {name = "name",     type = "String",       index = true},
+        {name = "age",      type = "Integer",      index = false},
+        {name = "lastseen", type = "Date",         index = true},
+        {name = "friends",  type = "List[String]", index = true},
+        {name = "geom",     type = "Point",        index = true, srid = 4326, default = true}
       ]
     }
-    {
-      type-name = "example-xml"
+    "example-xml" = {
       attributes = [
-        {name = "number", type = "Integer"}
-        {name = "color",  type = "String"}
-        {name = "weight", type = "Double"}
-        {name = "source", type = "String"}
-        {name = "geom", type = "Point", index = true, srid = 4326, default = true}
+        {name = "number", type = "Integer" }
+        {name = "color",  type = "String"  }
+        {name = "weight", type = "Double"  }
+        {name = "source", type = "String"  }
+        {name = "geom",   type = "Point", index = true, srid = 4326, default = true}
       ]
     }
-    {
-      type-name = "example-json"
+    "example-json" = {
       attributes = [
-        { name = "number", type = "Integer" }
-        { name = "color",  type = "String"  }
-        { name = "weight", type = "Double"  }
-        { name = "geom",   type = "Point"   }
+        {name = "number", type = "Integer"}
+        {name = "color",  type = "String" }
+        {name = "weight", type = "Double" }
+        {name = "geom",   type = "Point"  }
       ]
     }
-    {
-      type-name = "example-avro"
+    "example-avro" = {
       attributes = [
-        { name =  "dtg", type = "Date", default = true }
-        { name = "geom", type = "Point", index = "true", srid = 4326, default = true  }
+        {name = "dtg",  vtype = "Date", default = true }
+        {name = "geom", type = "Point", index = "true", srid = 4326, default = true }
```

Comment:
kind of picky, but we should try to be consistent with spaces before/after {} - right now we have them before but not after